app/views/charges/downgrade.html.erb

```diff
@@ -0,0 +1,4 @@
+<% if current_user.has_role? :premium %>
+    <h4>Click to downgrade to Standard user!</h4>
+    <%= link_to "Downgrade", method: :downgrade, class: 'btn btn-danger', data: { confirm: 'Are you sure you want to downgrade to standard user and relinquish power?' } %>
```

Comment:
Brittany,
Apologies, I'm still not getting it :(

So lets retrace my steps -

- I created this path in my layout file 
Premium User |  <%= link_to "Downgrade", downgrade_user_path %>

- I created the corresponding downgrade_user_html.erb for the same

-  I created the get route for the above 
get 'charges/downgrade' => 'charges#downgrade', :as => 'downgrade_charge'

Next,when I click on 'Downgrade' on the layout -

- It invokes a get and populates the downgrade.html.erb

- Clicking on this page will invoke a POST and call 'downgrade' method in the charges controller.

Now,when I remove the 'get' route and downgrade html,I need to invoke the 'downgrade' method in the controller directly from the layout file.And this is where I'm stuck.I cant figure out how to invoke the method without going through the route and using 'GET' to retrieve the html and then POSTing to the controller.